src/saving/SaveList.cs

```diff
@@ -188,6 +189,16 @@ private void OnNewSaveLoaded(string saveName)
         loadConfirmDialog.PopupCenteredMinsize();
     }
 
+    private void OnBrokenLoaded(string saveName)
+    {
+        saveToBeLoaded = saveName;
+
+        loadConfirmDialog.DialogText = "Thrive could not load this file.\n";
+        loadConfirmDialog.DialogText += "It seems like the save is broken.\n";
+        loadConfirmDialog.DialogText += "Do you want to try loading the save anyway?";
+        loadConfirmDialog.PopupCenteredMinsize();
```

Comment:
The text should be included in the tscn file instead of here as it isn't dynamically generated.

Reply:
But the other two dialogs also have their text set in code.
Should I move the text for the other dialogs as well?

Reply:
But don't those include dynamic elements? I think they include the save name / some number.

Reply:
I mean the other two dialogs right above my code

Reply:
I let that slide as the text is different based on if the attempted save load is newer or older, but it uses the same callbacks.
Static text should be defined in the scene file.

Reply:
But my dialog is also uses the same callbacks

Reply:
I guess this is what I get for letting one PR I'm not absolutely happy with through...
Anyway for your use case the popup title should be different instead of "Load incompatible save?" it should mention that the save info could not be loaded.
I really want different purpose popups to be separate. If you check out the tutorial system, I created a separate box for *each* tutorial popup.